mw_jwt.go

```diff
@@ -424,7 +463,7 @@ func (k *JWTMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _
 		if k.Spec.JWTSigningMethod == "rsa" {
```

Comment:
you introduced constant `RSASign` could you please switch the rest of code to use it?